keras/backend/mxnet_backend.py

```diff
@@ -3017,6 +3017,11 @@ def multi_hot_sparse_categorical_crossentropy(target, output, from_logits=False,
     >>>y_true_np2 = np.array([[1, 2], [0, 2],[0]])
     ```
     """
+    # TODO: remove version check after mxnet 1.3.1 stable release
```

Comment:
please create a tracking issue to remove this check

Reply:
tracked at https://github.com/awslabs/keras-apache-mxnet/issues/175